verticapy/connection/connect.py

```diff
@@ -125,6 +129,52 @@ def connect(section: str, dsn: Optional[str] = None) -> None:
         prev_conn.close()
     try:
         gb_conn.set_connection(vertica_connection(section, dsn), section, dsn)
+        print("Connected Successfuly!")
```

Comment:
you should look at the conf.get_option("print_info") var to see if you print or not the message.